ads/google/doubleclick.js

```diff
@@ -82,105 +67,99 @@ function doubleClickWithGpt(global, data, gladeExperiment, url) {
     dimensions = [[primaryWidth, primaryHeight]];
   }
 
-  loadScript(global, url, () => {
-    global.googletag.cmd.push(() => {
-      const googletag = global.googletag;
-      const pubads = googletag.pubads();
-      const slot = googletag.defineSlot(data.slot, dimensions, 'c')
+  loadScript(
+      global, 'https://www.googletagservices.com/tag/js/gpt.js', () => {
+        global.googletag.cmd.push(() => {
+          const googletag = global.googletag;
+          const pubads = googletag.pubads();
+          const slot = googletag.defineSlot(data.slot, dimensions, 'c')
           .addService(pubads);
 
-      if (gladeExperiment === GladeExperiment.GLADE_CONTROL) {
-        pubads.markAsGladeControl();
-      } else if (gladeExperiment === GladeExperiment.GLADE_OPT_OUT) {
-        pubads.markAsGladeOptOut();
-      }
-
-      if (data['experimentId']) {
-        const experimentIdList = data['experimentId'].split(',');
-        pubads.forceExperiment = pubads.forceExperiment || function() {};
-        experimentIdList &&
+          if (data['experimentId']) {
+            const experimentIdList = data['experimentId'].split(',');
+            pubads.forceExperiment = pubads.forceExperiment || function() {};
+            experimentIdList &&
```

Comment:
Let's wrap this in proper if or parens (e.g. (experimentIdList || []).forEach OR if (experimentIdList) experimentIdList.forEach